sentinel-extension/sentinel-annotation-aspectj/src/main/java/com/alibaba/csp/sentinel/annotation/aspectj/SentinelResourceAspect.java

```diff
@@ -76,7 +76,28 @@ public Object invokeResourceWithSentinel(ProceedingJoinPoint pjp) throws Throwab
             ContextUtil.exit();
         }
     }
-
+    
+    private String getResourceName(String resourceName, Method method) {
+        if(StringUtil.isNotBlank(resourceName)){
+            return resourceName;
+        }
+        StringBuilder buf = new StringBuilder(64);
+        buf.append(method.getDeclaringClass().getName())
```

Comment:
Here you can just use `MethodUtil.getMethodName(method)` to get its method representation, which will cache the method name for better performance.

```suggestion
return MethodUtil.getMethodName(method);
```